docs/intro/index.md

```diff
@@ -102,7 +102,7 @@ The core repositories have at least the following branches:
 The changes are recorded in [CHANGELOG.md](/CHANGELOG).
 
 ### Version support
-Before v8 every minor release of major releases are supproted for 1 year.
+Before v8 every minor release of major releases is supported for 1 year.
```

Comment:
I think the original sentence was incorrect.

```suggestion
The version policy changed in v8. Before v8, the last minor release of each major release series was supported for 1 year. This means that 7.11 (which was released in March 2021) is supported till March 2022.
As of v8, each minor release is supported for 1 year.
```

Reply:
Yeah, thankf for clarifying. This was a case where I wasn't so sure what the original meant.